src/csharp/TestApp/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.Spark.Extensions.Hadoop.FileSystem;
+using Microsoft.Spark.Sql;
+using System;
+
```

Comment:
Can we remove this file?

Reply:
Oops, didn't mean to commit that. Removed.